Richard-Fleming/Fort_MacArthur_Game
# Game over

Added a Game Over overlay that triggers upon having zero health or bullets.

Things to test:
- Open the game and shoot all your bullets (You should see the game over screen).
- Click back to menu button and you should go back to the main menu.
- Open the game again and let the enemies hit you until you have no health (You should see the game over screen).
- Click exit and the app should close.
